bazel/external/quiche.BUILD

```diff
@@ -4753,3 +4753,24 @@ envoy_cc_test(
         ":quic_platform",
     ],
 )
+
+envoy_cc_library(
+    name = "quiche_common_platform_lower_case_string_lib",
```

Comment:
For QUICHE libraries, they don't need "_lib" suffix in the target names.

Reply:
Thank you for the review.  I'm renaming quiche_common_platform_lower_case_string_lib and quiche_common_platform_header_policy_lib. 
 However, I'm not renaming quiche_lower_case_string_impl_lib or quiche_common_platform_default_quiche_platform_impl_header_policy_impl_lib, because there are no existing targets that end in _impl without the _lib suffix.